beluga-mqtt/src/lib.rs

```diff
@@ -48,6 +48,14 @@ impl<'a> MqttClientBuilder<'a> {
         }
     }
 
+    pub fn new_unencrypted() -> Self {
+        Self {
+            port: 1883,
```

Comment:
Let's change the `port` to be `Option<u16>` and then in `.build()` we can use the appropriate default value depending on whether we're using TLS or raw TCP.
With that change we also no longer need this function.